e2e-tests/commands/insurance/assert-alternative-currency-form.js

```diff
@@ -21,22 +40,24 @@ const { CURRENCY: { CURRENCY_CODE, ALTERNATIVE_CURRENCY_CODE } } = INSURANCE_FIE
  * - HINT - text for hint
  * - ALTERNATIVE_CURRENCY_TEXT - text for "another currency" input
  * - ALTERNATIVE_CURRENCY_CODE - fieldId for "another currency" input
+ * - ERRORS - error message object
  */
 const assertAlternativeCurrencyForm = ({
   FIELD_ID = CURRENCY_CODE,
   LEGEND,
   HINT,
   ALTERNATIVE_CURRENCY_TEXT,
   ALTERNATIVE_CURRENCY_FIELD_ID = ALTERNATIVE_CURRENCY_CODE,
+  ERRORS,
 }) => ({
   legend: () => cy.checkText(field(FIELD_ID).legend(), LEGEND),
   hint: () => cy.checkText(field(FIELD_ID).hint(), HINT),
   radios: () => {
-    const { option: option1 } = radios(FIELD_ID, EUR.isoCode);
-    const { option: option2 } = radios(FIELD_ID, GBP.isoCode);
-    const { option: option3 } = radios(FIELD_ID, USD.isoCode);
-    const { option: option4 } = radios(FIELD_ID, JPY.isoCode);
-    const { option: option5 } = radios(ALTERNATIVE_CURRENCY_FIELD_ID);
+    const option1 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID, EUR.isoCode);
+    const option2 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID, GBP.isoCode);
+    const option3 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID, USD.isoCode);
+    const option4 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID, JPY.isoCode);
+    const option5 = whichRadio(FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID);
```

Comment:
I think the original cleaner 🤔 Here we're passing `ALTERNATIVE_CURRENCY_FIELD_ID`  4 times for it not to be used?